universal-application-tool-0.0.1/app/views/LoginForm.java

```diff
@@ -55,6 +61,57 @@ public Content render(Http.Request request, Optional<String> message) {
                   routes.CallbackController.callback(FakeAdminClient.CLIENT_NAME).url())));
     }
 
-    return layout.htmlContent(bodyTag);
+    return layout.htmlContent(head(layout.tailwindStyles()), demoBanner(), bodyTag);
+  }
+
+  public ContainerTag demoBanner() {
+    return div()
+        .withClass(Styles.BG_RED_600)
+        .with(
+            div()
+                .withClasses(
+                    Styles.MAX_W_7XL, Styles.MX_AUTO, Styles.PY_3, Styles.PX_3, "sm:px-6 lg:px-8")
+                .with(
+                    div()
+                        .withClasses(
+                            Styles.FLEX,
+                            Styles.ITEMS_CENTER,
+                            Styles.JUSTIFY_BETWEEN,
+                            Styles.FLEX_WRAP)
+                        .with(
+                            div()
+                                .withClasses(
+                                    Styles.W_0, Styles.FLEX_1, Styles.FLEX, Styles.ITEMS_CENTER)
+                                .with(
+                                    span()
+                                        .withClasses(
+                                            Styles.FLEX,
+                                            Styles.P_2,
+                                            Styles.ROUNDED_LG,
+                                            Styles.BG_RED_800)
+                                        .with(
+                                            new ContainerTag("svg")
+                                                .withClasses(
+                                                    Styles.H_6, Styles.W_6, Styles.TEXT_WHITE)
+                                                .attr("xmlns", "http://www.w3.org/2000/svg")
+                                                .attr("fill", "none")
+                                                .attr("viewBox", "0 0 24 24")
+                                                .attr("stroke", "currentColor")
+                                                .attr("aria-hidden", "true")
+                                                .with(
+                                                    new ContainerTag("path")
+                                                        .attr("stroke-linecap", "round")
+                                                        .attr("stroke-linejoin", "round")
+                                                        .attr("stroke-width", "2")
+                                                        .attr("d", Icons.LOGIN_BANNER_PATH))),
+                                    p().withClasses(
+                                            Styles.ML_3,
+                                            Styles.FONT_MEDIUM,
+                                            Styles.TEXT_WHITE,
+                                            Styles.TRUNCATE)
+                                        .with(
+                                            span(BANNER_TEXT).withClasses("md:hidden"),
```

Comment:
I don't think you need both spans. 

try `withClasses(StylesUtils.responsiveMedium(Styles.INLINE)`

...but that still shouldn't be necessary as span should default to inline.

Reply:
Thanks for pointing out. I was just blindly copying the styles over.

Reply:
Done.